DQM/L1TMonitor/plugins/SealModule.cc

```diff
@@ -112,9 +115,3 @@ DEFINE_FWK_MODULE( L1GtHwValidation);
 
 #include "DQM/L1TMonitor/interface/L1ExtraDQM.h"
 DEFINE_FWK_MODULE(L1ExtraDQM);
-
```

Comment:
Why did you remove these lines?

Reply:
These two still need to be added back. Compare to this file https://github.com/cms-sw/cmssw/blob/master/DQM/L1TMonitor/plugins/SealModule.cc